extensions/positron-python/src/test/pythonEnvironments/nativePythonFinder.unit.test.ts

```diff
@@ -17,6 +17,7 @@ import * as workspaceApis from '../../client/common/vscodeApis/workspaceApis';
 
 // --- Start Positron ---
 // re-enable when Native Finder is used for discovery
+// TODO: add test for python.interpreters.include here once we switch to Native Finder
```

Comment:
this is fine for now, I'll log an issue to turn these back on (probably for the next milestone)